src/views/Home/components/NewsCard/index.jsx

```diff
@@ -12,7 +13,7 @@ const NewsCard = () => {
   }, []);
 
   return (
-    <Card>
+    <Card className={styles.card}>
```

Comment:
Is this necessary? The Card should have default styles, so this is either irrelevant or else you're changing the styles of this card specifically, which I don't think is in the scope of this PR.

Reply:
Yep, it is necessary, the card by default gets a color from the MUI palette, which isn't changed by out theme palette.  For all of the cards, I'm having to add a className to them to reference the CSS theme variables for their background colors...

Reply:
Hmm, that seems to me like you might've configured the dark theme wrong then. As I understand it, MUI has a default dark theme which should effect all components when you switch themes, so you don't need to, for example, restyle all MUI components (which kinda defeats the purpose of using MUI in the first place).

Adding a class to EVERY `Card` component in the codebase is not an acceptable solution, so this might need further investigation.

Reply:
So, MUI does have a default dark theme, which works, but, at least the way I set up the themes, our theme doesn't override the card color from MUI.  I don't know if you have any idea what I might need to change to make our dark mode override the MUI properly.

One thing I do know, a lot of MUI components like buttons have color props that are set to MUI them setting, like color='primary,' and they DO change with our defined theme colors...  For testing purposes, I set all of the dark theme colors to yellow, so we can easily see which components have been changed to reference the correct theme CSS variables...  Buttons which have MUI color props do turn yellow in dark mode, but the cards stay the default MUI dark mode dark gray color.

I don't know if that helps you understand this at all @EjPlatzer , I'll try to read up on how all these themes interact, because I clearly don't understand it fully...

Reply:
In the meantime, I have been changing other cards in this way, which seems to work fine, but maybe I don't understand the problem with this.  
Is the problem that defining a CSS class for them overrides all the default MUI styles, thus defeating the purpose?  
I guess I would expect that defining a CSS class only overrides the MUI styles for CSS properties that we include manually, but that might be a faulty assumption on my part...

Reply:
Maybe a clearer way to say this is that, rather than overriding the MUI default again each time we use the MUI component (`Card` in this case), we should override the component's styling once at the theme level so that every `Card` across the entire site automatically gets overridden in a consistent way without us having to do any extra work in each place that `Card` is used.

I think perhaps the piece that you're missing is that all of the MUI theme is configurable, not just the palette. So if you want `Card` to have different styles, you can declare the `Card` styles for each theme mode inside the base theme, and it will apply to every `Card`.

The question I would ask is - do you need to change the `Card` styling? You mentioned you're using some yellow colors just for testing, but you only applied those colors to the palette colors, right? So it's expected that the MUI components that don't reference our palette colors are still using the default dark mode theme. Is there something wrong with the default dark mode theme? If so, you should change it at the theme level as outlined above. If not, you shouldn't change it all - just let the theme be as is.

Does that make sense?

Reply:
Ah, got it, that is helpful.  I think we probably want the cards to reference our palette colors, because we'll probably want to customize the color it chooses, especially in dark mode, rather than the default... but yes, that makes total sense that there is an easier way that I didn't understand, thanks!